<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newGridDlg</class>
 <widget class="QDialog" name="newGridDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="newGridWgt" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}

QPushButton{
	
	font: 7pt &quot;MS Shell Dlg 2&quot;;
	border-radius: 10px;
	border: solid;
	border-width: 2px;
	border-color: rgb(255, 255, 255);
	background-color: rgb(63, 63, 63);
}
QPushButton:pressed{
	background-color: rgb(127, 127, 127);
}


#mainWgt{
	border-radius: 15px;
	border: solid;
	border-width: 1px;
	border-color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QLabel" name="newGridLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Crea Nouva Rete</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nameLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome rete:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sourceVLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>V rete esterna</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sourceVDsb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> kV</string>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QPushButton" name="createPb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelPb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Annulla</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
